CDP4Composition/Services/FavoritesService/FavoritesService.cs

```diff
@@ -55,6 +57,11 @@ public class FavoritesService : IFavoritesService
         /// </summary>
         private static Logger logger = LogManager.GetCurrentClassLogger();
 
+        /// <summary>
+        /// The INJECTED <see cref="IServerExceptionService"/>
+        /// </summary>
+        private IServerExceptionService serverExceptionService = ServiceLocator.Current.GetInstance<IServerExceptionService>();
```

Comment:
not so happy about this

Reply:
I know. Was the fastest way to check if everything worked. Otherwise every VM (Browser, Dialog and probably also Row) should have an updated constructor and should be created accordingly.
But maybe adding "plugable" validation  process handling (using IME side implementations of the new IExceptionHandler, that would be moved to SDK, together with IServerExceptionService and ServerExceptionService) at ISession level will also work and would give a clearer result.